src/libs/IOUUtils.ts

```diff
@@ -116,16 +116,9 @@ function isValidMoneyRequestType(iouType: string): boolean {
         CONST.IOU.TYPE.PAY,
         CONST.IOU.TYPE.TRACK,
         CONST.IOU.TYPE.INVOICE,
+        CONST.IOU.TYPE.GLOBAL_CREATE,
```

Comment:
Do we actually need to add this? Since we're not adding a new type of IOU, just a new path to the old types (i.e. REQUEST/SUBMIT/TRACK)

Reply:
I need to add it there, otherwise we'll have a Not Found page due to `AccessOrNotFoundWrapper` blocking the access of this new IOU type.